Civi/Token/StandardFilters.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+namespace Civi\Token;
+
+/**
+ * This class is a collection of CiviMail filter functions. For example, consider this message:
```

Comment:
opps - I just noticed the comment incorrectly references `CiviMail` - which is just one consumer of tokens

Reply:
OK @totten I put up a PR to fix it https://github.com/civicrm/civicrm-core/pull/25730